website/integrations/services/glitchtip/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Glitchtip
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Glitchtip
+
+> Bugs are inevitable in web development. The important thing is to catch them when they appear. With GlitchTip, you can rest easy knowing that if your web app throws an error or goes down, you will be notified immediately. GlitchTip combines error tracking and uptime monitoring in one open-source package to keep you and your team fully up-to-date on the status of your projects.
+>
+> -- https://glitchtip.com/documentation
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `glitchtip.company` is the FQDN of the Glitchtip install.
+-   `authentik.company` is the FQDN of the authentik install.
+
+## authentik Configuration
```

Comment:
```suggestion
## authentik configuration
```

Reply:
@tanberry I copied this (uppercase C for configuration) from one of the other ones: https://github.com/search?q=repo%3Agoauthentik%2Fauthentik+%23%23+%22authentik+Configuration%22&type=code

In general it seems like there's inconsistencies in the headings and layouts of these service integration pages. Maybe they should all use a standard template?

Reply:
Yep, I was thinking we needed a template (the inconsistencies between the integrations are pretty wild) and then someone pointed out that we already DO have a template for integrations, haha. It is just rather hard to find: `https://github.com/goauthentik/authentik/blob/main/website/integrations/_template/service.md`

I'll find places to add a link to the tempalte, and in general try to get the word out that people contributing Integrations docs should use the template, and not copy form another (possibly incorrect) integration page. Thank you for making me dig into this more. :-)

We also do have a very very baby Style Guide for Docs... Needs a lot of work though... I'll get to it eventually.

Reply:
@tanberry I saw that template but didn't it because it looked pretty different to most of the other service pages I looked at 😅